Makefile

```diff
@@ -1,7 +1,7 @@
 REGISTRY?=directxman12
 IMAGE?=k8s-prometheus-adapter
 TEMP_DIR:=$(shell mktemp -d)
-ARCH?=amd64
+ARCH?=$(shell go env GOARCH)
```

Comment:
I would prefer not to deviate from upstream to minimize maintenance churn. Note that the `?=` operator sets the variable only if it is unset. Invoking `make ARCH=${GOARCH}` should do the trick here.

Reply:
Would a better alternative be to update the Dockerfiles to call the Makefile with GOARCH as an argument?

Thanks!

Reply:
After sending the exact same PR!, I agree with @s-urbaniak as updating the Dockerfile looks more cleaner solution. Can this PR be updated to add `make ARCH={GOARCH}` to Dockerfile? @dbenoit17